identity-service/sequelize/migrations/20190221071525-users.js

```diff
@@ -23,7 +23,7 @@ module.exports = {
       isConfigured: {
         type: Sequelize.BOOLEAN,
         allowNull: false,
-        defaultValue: false
+        defaultValue: true
```

Comment:
@SidSethi i don't think we need this. we shouldn't retroactively change migrations